### DISCUSSION TITLE
File upload to Asset, modifies the file name?

### DISCUSSION THREAD

**Discussion by barau83** [snipe/snipe-it#10256] [Q&A] [answered]
Hi, not sure if this has been covered before, but why does a file's name change when uploaded to an Asset (under the File tab). For example, a pdf named 'Invoice.pdf' is renamed to 'hardware-89-U8lUwC18-invoice.pdf'. Is this normal?

![Capture](https://user-images.githubusercontent.com/64520507/139624757-7aaeb055-e4a5-4dcc-a770-399eb17806d4.JPG)

**Comment by uberbrady** [comment, marked as answer]
Yes, that's normal. We have to enforce uniqueness on filenames so that people can't inadvertently (or advertently) overwrite an uploaded file.
